diff --git a/lib/utils.js b/lib/utils.js
index 4a6734f78af172f4d96c41f5cb6be80623d9f466..d283e09c904fc79112f076a7da1c098b724ea334 100644
--- a/lib/utils.js
+++ b/lib/utils.js
@@ -8,6 +8,7 @@ export async function _importRelative(installDir, packageWithPath) {
     return await import(pathToFileURL(_resolveRelative(installDir, packageWithPath)).href);
 }
 export function _resolve(packageWithPath) {
+    const require = createRequire(import.meta.url)
     return require.resolve(packageWithPath);
 }
 export function _resolveRelative(installDir, packageWithPath) {